### PR TITLE
Hostname-based routing, participant form updates

### PR DESCRIPTION
**Pull request by ashtonmeuser**
Participant form is visible at `/participant-form`. It is also shown at `/` depending on hostname. For now, a hostname of `hcapparticipants.gov.bc.ca` will show the participant form at the base path.

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -315,6 +315,7 @@ app.post(`${apiBaseUrl}/participants`,
 
     const participant = {
       ...req.body,
+      formVersion: 'v1',
```
Comment (fwkendall) [inline server/server.js:318]: 🙌

File: client/src/components/participant-form/Fields.js [hunk 2]
```diff
@@ -40,7 +40,7 @@ export const Fields = ({ isDisabled }) => {
         </Grid>
         <Grid item xs={12}>
           <Typography>
-            <b>* Are you a Canadian citizen, permanent resident or <Link href="https://www.canada.ca/en/immigration-refugees-citizenship/services/work-canada/permit/temporary/eligibility.html" target="__blank" rel="noreferrer noopener">otherwise legally eligible to work in Canada?</Link></b>
+            <b>* Are you a Canadian citizen, permanent resident or <Link href="https://www.canada.ca/en/immigration-refugees-citizenship/services/work-canada.html" target="__blank" rel="noreferrer noopener">otherwise legally eligible to work in Canada?</Link></b>
```
Comment (fwkendall) [inline client/src/components/participant-form/Fields.js:43]: Nice catch!

**Review by fwkendall — APPROVED** [review]

File: client/src/components/generic/Header.js [hunk 3]
```diff
@@ -70,7 +70,7 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-export const Header = () => {
+export const Header = ({hideEmployers=false}) => {
```
Comment (Zenen-freshworks) [inline client/src/components/generic/Header.js:73]: Nitpick: I think that our styling standards suggest spaces on the edges of ({ this="kind of statement" })

File: client/src/routes/index.js [hunk 4]
```diff
@@ -22,7 +22,7 @@ const EmployerForm = lazy(() => import('../pages/public/EmployerForm'));
 const Login = lazy(() => import('../pages/public/Login'));
 const ParticipantConfirmation = lazy(() => import('../pages/public/ParticipantConfirmation'));
 const EmployerConfirmation = lazy(() => import('../pages/public/EmployerConfirmation'));
-const KeycloackRedirect = lazy(() => import('../pages/public/Keycloak'));
+const KeycloakRedirect = lazy(() => import('../pages/public/Keycloak'));
```
Comment (Zenen-freshworks) [inline client/src/routes/index.js:25]: thank you!!

**Review by Zenen-freshworks — APPROVED** [review]

**Pull request merged**